content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -254,3 +212,43 @@ authentication process. If `GDRIVE_CREDENTIALS_DATA` is set, the
 `gdrive_user_credentials_file` value (if provided) is ignored.
 
 > Please note our [Privacy Policy (Google APIs)](/doc/user-guide/privacy).
+
+## Using service accounts
```

Comment:
This section is unchanged and only moved to the bottom

Reply:
Not exactly unchanged: It's missing the `GDRIVE_CREDENTIALS_DATA` paragraph, but that's already in **Authentication** which is now before this section. Is this the intention? (To have it only above)

Reply:
Right, especially with the new paragraph about specifying that the "Authorization" section is about user accounts, it seems unclear to me that `GDRIVE_CREDENTIALS_DATA` can also be used for service accounts.

Reply:
> Not exactly unchanged:

good catch, that's a bug, I'll fix it. It's not intentional at all.